builds/runtimes/pypy-5.1.0

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
Oh, well, I guess we should send a patch to heroku about this, not just change this in our repo.